package.json

```diff
@@ -94,18 +95,30 @@
   "APIMethod": "stub",
   "proxyURL": "http://localhost:8000",
   "scripts": {
-    "test": "npm run jest -- tests/js/spec",
-    "test-ci": "npm run test -- --runInBand",
-    "test-watch": "npm run test -- --watch",
-    "jest": "NODE_ENV=test node_modules/.bin/jest",
-    "jest-debug": "NODE_ENV=test node --inspect-brk node_modules/.bin/jest --runInBand",
+    "test": "node scripts/test.js --env=jsdom",
+    "test-ci": "node scripts/test.js -- --runInBand --ci --testResultsProcessor=jest-junit",
```

Comment:
not that it matters much but shouldn't these all work if calling `npm run test`?